src/data/sym/basic.lean

```diff
@@ -150,4 +159,111 @@ instance inhabited_sym' [inhabited α] (n : ℕ) : inhabited (sym' α n) :=
 
 end inhabited
 
+instance has_zero {α : Type*} : has_zero (sym α 0) := ⟨⟨0, rfl⟩⟩
+instance has_emptyc {α : Type*} : has_emptyc (sym α 0) := ⟨0⟩
+
+instance subsingleton {α : Type*} {n : ℕ} [g : subsingleton α] : subsingleton (sym α n) :=
```

Comment:
```suggestion
instance has_zero (α : Type*) : has_zero (sym α 0) := ⟨⟨0, rfl⟩⟩
instance has_emptyc (α : Type*) : has_emptyc (sym α 0) := ⟨0⟩

instance subsingleton (α : Type*) (n : ℕ) [g : subsingleton α] : subsingleton (sym α n) :=
```

Reply:
I don't think this really matters, and haven't seen much of a pattern regarding whether we do this or not.

More usual though would be to add `variables {α : Type*}` somewhere rather than repeating it throughout this PR.